.devcontainer/.devcontainer.json

```diff
@@ -0,0 +1,26 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ruby
+{
+	"name": "solid_queue",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "ghcr.io/rails/devcontainer/images/ruby:3.3.4",
```

Comment:
I wonder if we should publish 3.3 tag. But I published 3.3.5 yesterday if you want to update this.